include/wx/osx/setup0.h

```diff
@@ -1622,6 +1622,13 @@
 // make sure we have the proper dispatcher for the console event loop
 #define wxUSE_SELECT_DISPATCHER 1
 #define wxUSE_EPOLL_DISPATCHER 0
+
+// set to 1 if you have older code that still needs icon refs
+#define wxOSX_USE_ICONREF 0
```

Comment:
If we intend to keep these options (i.e. if they're not going to be removed before/soon after this PR is merged), we also need to check for them in `include/wx/osx/chkconf.h`.